components/InputComponent.tsx

```diff
@@ -2,19 +2,26 @@ import { TextInput, StyleSheet } from "react-native";
 
 interface InputComponentProps {
   placeholder?: string;
-  onChangeText?: (text: string) => void;
+  onChangeText: (text: string) => void;
+  value: string;
+  onSubmitEditing: () => void;
 }
 
 export function InputComponent({
   placeholder,
   onChangeText,
+  value,
+  onSubmitEditing,
 }: InputComponentProps) {
   return (
     <TextInput
       style={styles.input}
       placeholder={placeholder ? placeholder : ""}
       placeholderTextColor={placeholder ? "#747272" : "#231F20"}
       onChangeText={onChangeText}
+      value={value}
+      returnKeyType="done" // Change the return key type to "done" on mobile keyboard
```

Comment:
Kanske "add" istället så fattar man att den läggs till direkt om man trycker där?

Reply:
finns inte det alternativet, dessa finns:
"go": Displays a "Go" button.
"next": Displays a "Next" button.
"search": Displays a "Search" button.
"send": Displays a "Send" button.
"default": Displays the default return key.
"emergency-call": Displays an "Emergency Call" button (iOS only).
"google": Displays a "Google" button (Android only).
"join": Displays a "Join" button.
"route": Displays a "Route" button.
"yahoo": Displays a "Yahoo" button (Android only).
"done": Displays a "Done" button.